packages/machine/src/protocol/update.ts

```diff
@@ -85,14 +91,20 @@ export const UPDATE_PROTOCOL: ProtocolExecutionFlow = {
     // Sign the same state update yourself
     Opcode.OP_SIGN,
 
+    (message: ProtocolMessage, context: Context) => {
+      context.finalCommitment = context.commitments[0].transaction([
+        context.signatures[0],
+        message.signature!
+      ]);
```

Comment:
For code clarity could we name these values?

e.g.,

```ts
    function generateSignedCommitment(message: ProtocolMessage, context: Context) => {
        const setStateCommitment = context.commitments[0] as SetStateCommitment;
        const mySignature = context.signatures[0];
        const counterpartySignature = message.signature as Signature;
	      context.finalCommitment = setStateCommitment.transaction([
            mySignature, 
            counterpartySignature
        ]);
    }

Reply:
good idea